OpenRA.Mods.Common/Widgets/ConfirmationDialogs.cs

```diff
@@ -15,13 +15,25 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public static class ConfirmationDialogs
 	{
-		public static void PromptConfirmAction(string title, string text, Action onConfirm, Action onCancel = null, string confirmText = null, string cancelText = null)
+		public static void PromptConfirmAction(
+			string title,
+			string text,
+			Action onConfirm,
+			Action onCancel = null,
+			string confirmText = null,
+			string cancelText = null,
+			World world = null,
+			Action onRestart = null,
```

Comment:
If I change the action and strings around all current calls to PromptConfirmAction will require a null before the labels because they do not use restart.